### DISCUSSION TITLE
Bugzilla migration to GitHub issues?

### DISCUSSION THREAD

**Discussion by Marcono1234** [eclipse-jdt/eclipse.jdt.core#2773] [Q&A] [answered]
What is the current status regarding https://bugs.eclipse.org for the JDT project?

I just noticed an issue which is only reported on Bugzilla so far but not as GitHub issue (the issue is https://bugs.eclipse.org/bugs/show_bug.cgi?id=577790).
- Should I comment on the Bugzilla issue?
- Are the Bugzilla issues still being looked at?
- Or should I just create a new GitHub issue here and refer to the Bugzilla issue?
- And are there plans to auto-migrate the Bugzilla issues here?

**Comment by iloveeclipse** [comment, marked as answer]
* you can
* very seldom
* yes
* no

**Reply by Marcono1234** [reply]
Ok thanks, I have created #2778 now as clone.
